lab_1_classify_by_unigrams/main.py

```diff
@@ -4,104 +4,71 @@
 """
 
 
-def tokenize(text: str) -> list[str] | None:
```

Comment:
Please do not delete docstrings!
Bring them back for all functions. Thank you!

Reply:
I brought all strings back

Reply:
Done

Reply:
Thanks! 
But as you can see in the `files changes` section of github, the docstrings were still altered a bit. 
I would be grateful If you removed this change. In most cases, it is just indentation.

Reply:
I tried to do it once more. But I don't know what else can be done